integration/apps/hpl_mkl/README.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+
+From https://www.netlib.org/benchmark/hpl/:
+
+    HPL is a software package that solves a (random) dense linear
+    system in double precision (64 bits) arithmetic on distributed-memory
+    computers. It can thus be regarded as a portable as well as freely
+    available implementation of the High Performance Computing Linpack Benchmark.
+
+This AppConf runs the MKL version of HPL, which is optimized for Intel
+processors.
+
+Testing has been done with MKL 19.0.4.243 and impi 2019.4.243 (dynamically linked).
+IMPI is required for the best performance.
+
+The --perc-dram option passed to the run scripts that call this banchmark
```

Comment:
Is this value from 0.0 to 1.0, or 0 to 100?

Reply:
I clarified it and will push update.